src/templates/includes/navbar.pug

```diff
@@ -37,11 +37,10 @@ nav.navbar.is-white(role="navigation" aria-label="main navigation")
       div.navbar-end
         div.navbar-item
           div.field
-            div.control.has-icons-left
+            form.control.has-icons-left(onsubmit="search()")
               input.input.is-rounded.searchbar(
                 type="text"
                 placeholder="Search the HNS Blockchain"
-                maxlength="64"
-                onkeypress="catchEnter(event)")
+                maxlength="63")
```

Comment:
funnily enough, we missed the correct value by 1. https://www.google.com/search?q=max+length+of+a+tld&oq=max+length+of+a+tld&aqs=chrome..69i57j0l5.3534j1j1&sourceid=chrome&ie=UTF-8